oper8/constants.py

```diff
@@ -41,7 +41,7 @@
 
 # The name of the annotation used to indicate the internal name of each
 # oper8-managed resource
-INTERNAL_NAME_ANOTATION_NAME = "oper8.org/internal-name"
+INTERNAL_NAME_ANNOTATION_NAME = "oper8.org/internal-name"
```

Comment:
Thank you for fixing this! I think we need to make an alias (with a comment) to the old misspelled name to avoid a breaking change since some derived modules use this directly.

Reply:
Do you mean something like this [82fe425](https://github.com/IBM/oper8/pull/133/commits/82fe42523ed8aa29f181351467163e5d0faa4809)?

Reply:
Perfect!